src/store/cache/proposals/hooks.ts

```diff
@@ -25,8 +25,11 @@ export const getAllProposals = (state: AppState) =>
   state.cache.proposals.allProposals
 export const getResolvedProposals = (state: AppState) => {
   const allProposals = getAllProposals(state)
+  // need to reverse the proposals to return in reverse-chronological order
   return (
-    state.cache.proposals.resolvedProposals?.map(p => allProposals[p]) ?? null
+    state.cache.proposals.resolvedProposals
```

Comment:
Nit, I would do this once in the "reducer" rather than on the "selector" which runs every time the component renders.

This would be in store/proposals/slice.ts on line 55

Reply:
will fix in a min